dockerfiles/components/clarinet-dev.dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /src
 
 RUN apt update && apt install -y ca-certificates pkg-config libssl-dev
 
-RUN rustup update 1.67.0 && rustup default 1.67.0
+RUN rustup update 1.71.0 && rustup default 1.71.0
```

Comment:
This file is not used _yet_
It was introduced in #1104 and rightfully used v1.67.0, reflecting what was set in rust-toolchain at the time